overlord/snapstate/snapmgr.go

```diff
@@ -556,6 +556,29 @@ func (m *SnapManager) RefreshSchedule() (string, bool, error) {
 	return m.autoRefresh.RefreshSchedule()
 }
 
+// EnsureAutoRefreshesAreDelayed will delay refreshes for the specified amount
+// of time, as well as return any active auto-refresh changes that are currently
+// not ready so that the client can wait for those.
```

Comment:
Nice! It looks like we can use this one also when we implement: `snap refresh --hold=2h` in the future, yes?

Reply:
yeah good point actually that happened entirely by accident :smile: